ansible-playbook/roles/gophish/tasks/main.yml

```diff
@@ -80,29 +133,48 @@
 - name: Unzip gophish file.
   unarchive:
     src: "/home/{{ gophish_user }}/gophish.zip"
-    dest: "/home/{{ gophish_user }}/gophish"
+    dest: "/home/{{ gophish_user }}/gophish_deploy"
     remote_src: True  # File is on target server and not locally.
     owner: "{{ gophish_user }}"
     group: "{{ gophish_user }}"
 
 - name: Change ownership of Gophish folder and files.
   file:
-    path: /home/{{ gophish_user }}/gophish
+    path: /home/{{ gophish_user }}/gophish_deploy
     owner: "{{ gophish_user }}"
     group: "{{ gophish_user }}"
     recurse: True
 
-- name: Allow gophish binary to bind to privileged ports using setcap.
-  shell: setcap CAP_NET_BIND_SERVICE=+eip /home/{{ gophish_user }}/gophish/gophish
+- name: Ensure gophish binary is allowed to bind to privileged ports using setcap
+  capabilities:
```

Comment:
As part of this, could we add `libcap2-bin` to the default `{{ install_packages }}`? In a vanilla Ubuntu system, I got an error that `getcap` wasn't available.

Reply:
You're right, I also added it :)